draft-barnes-acme.md

```diff
@@ -1621,37 +1628,25 @@ token (required, string):
 least 128 bits of entropy, in order to prevent an attacker from guessing it.
 It MUST NOT contain any characters outside the URL-safe Base64 alphabet.
 
-~~~~~~~~~~
-{
-  "type": "dvsni",
-  "token": "evaGxfADs6pSRb2LAv9IZf17Dt3juxGJ-PCt92wr-oA"
-}
-~~~~~~~~~~
-
-In response to the challenge, the client uses its account private key to sign a
-JWS over a JSON object describing the challenge.  The validation object covered
-by the signature MUST have the following fields and no others:
-
-type (required, string):
-: The string "dvsni"
-
-token (required, string):
-: The token value from the server-provided challenge object
+authorizedKeys (required, string):
```

Comment:
It's really odd for the server-provided challenge to already have an `authorizedKeys` field. Can you explain why?